File: sdk/java-sdk/src/main/java/com/evernym/verity/sdk/exceptions/ProvisionTokenException.java
package com.evernym.verity.sdk.exceptions;

/**
 *  Exception class used express an invalid provision token.
 */
public class ProvisionTokenException extends VerityException {
    /**
     * Constructs a new ProvisionTokenException with the given message.
     *
     * @param message the given message for the cause of the exception.
     *
     * @see VerityException#VerityException(String)
     */
    public ProvisionTokenException(String message) {super(message);}

    /**
     * Constructs a new ProvisionTokenException with the given message and cause.
     *
     * @param message the given message for the cause of the exception.
     * @param cause the given cause (Throwable) for the exception.
     *
     * @see VerityException#VerityException(String, Throwable)
     */
    public ProvisionTokenException(String message, Throwable cause) {super(message, cause);}
}

File: sdk/dotnet-sdk/src/VeritySDK/Exception/ProvisionTokenException.cs
using System;

namespace VeritySDK
{
    /// <summary>
    /// Exception class used express an invalid provision token.
    /// </summary>
    public class ProvisionTokenException : VerityException
    {
        /// <summary>
        /// Constructs a new ProvisionTokenException with the given message.
        /// </summary>
        /// <param name="message">the given message for the cause of the exception</param>
        public ProvisionTokenException(String message) : base(message) { }

        /// <summary>
        /// Constructs a new ProvisionTokenException with the given message and cause.
        /// </summary>
        /// <param name="message">the given message for the cause of the exception.</param>
        /// <param name="cause">the given cause(Throwable) for the exception.</param>
        public ProvisionTokenException(String message, System.Exception cause) : base(message, cause) { }
    }

}
